File: Python_Fundamental/Functions/odd_and_even_sum.py
def odd_and_even_sum(number):
    sum_even = 0
    sum_odd = 0
    new_list = list(number)
    for element in new_list:
        int_element = int(element)
        if int_element % 2 == 0:
            sum_even += int_element
        else:
            sum_odd += int_element
    print(f"Odd sum = {sum_odd}, Even sum = {sum_even}")


current_string = input()
odd_and_even_sum(current_string)

File: JavaScript/Functions_and_statements/odd_and_even_sum.js
function oddAndEvenSum(number) {
    let oddArray = []
    let evenArray = []
    let sumOfEvenArray = 0
    let sumOfOddArray = 0

    number = String(number)
    // console.log(typeof(number))
    for (let i = 0; i < number.length; i++) {
        const element = number[i];
        // console.log(element)
        if (element % 2 === 0) {
            evenArray.push(element)
        } else {
            oddArray.push(element)
        }
    }
    for (const iterator of oddArray) {
        currentNum = Number(iterator)
        // console.log(currentNum)
        sumOfOddArray += currentNum
    }
    for (const iterator of evenArray) {
        currentNum = Number(iterator)
        sumOfEvenArray += currentNum
    }
    // console.log(evenArray)
    // console.log(oddArray)
    // console.log(sumOfEvenArray)
    // console.log(sumOfOddArray)
    console.log(`Odd sum = ${sumOfOddArray}, Even sum = ${sumOfEvenArray}`)
}

oddAndEvenSum(3495892137259234)